frontend/test/metabase/lib/formatting.unit.spec.js

```diff
@@ -95,15 +95,15 @@ describe("formatting", () => {
       });
       it("shouldn't display small numbers as 0", () => {
         expect(formatNumber(0.1, { compact: true })).toEqual("0.1");
-        expect(formatNumber(-0.1, { compact: true })).toEqual("-0.1");
+        expect(formatNumber(-0.1, { compact: true })).toEqual("−0.1");
         expect(formatNumber(0.01, { compact: true })).toEqual("0.01");
-        expect(formatNumber(-0.01, { compact: true })).toEqual("-0.01");
+        expect(formatNumber(-0.01, { compact: true })).toEqual("−0.01");
```

Comment:
D3 v6 uses the minus sign instead of the hyphen-minus for negative numbers. That was a fun bug to track. https://github.com/d3/d3-format/issues/62

U+002D: HYPHEN-MINUS {hyphen, dash; minus sign}
U+2212: MINUS SIGN